src/components/ha-selector/ha-selector-color-temp.ts

```diff
@@ -46,14 +46,12 @@ export class HaColorTempSelector extends LitElement {
 
   static styles = css`
     ha-labeled-slider {
-      --ha-slider-background: -webkit-linear-gradient(
-        var(--float-end),
+      --ha-slider-background: linear-gradient(
+        to right,
         rgb(255, 160, 0) 0%,
         white 50%,
         rgb(166, 209, 255) 100%
       );
-      /* The color temp minimum value shouldn't be rendered differently. It's not "off". */
-      --paper-slider-knob-start-border-color: var(--primary-color);
     }
```

Comment:
This isn't working anymore, hence the changes. Might need some extra work to let it work as it used to.

Reply:
This one is incorrect as well, the ltr direction styles were not set for gallery.